extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -568,7 +576,9 @@ export class AmpStoryPage extends AMP.BaseElement {
         )
       );
     }
-    this.muteAllMedia();
+    this.muteAllMedia().then(() => {
+      this.backgroundAudioUpgradeResolveFn_();
```

Comment:
We create the background audio with a `muted` attribute so maybe we can save a tiny bit of time by resolving the promise earlier